### ISSUE TITLE
Disable cache clearing when product category term is updated not working (WC REST API Request)

### ISSUE THREAD

**Issue by GitHub-Mike** [wp-media/wp-rocket#6636]
I use a snippet from the [Disable Cache Clearing Helper Plugin](https://github.com/wp-media/wp-rocket-helpers/blob/master/cache/wp-rocket-no-cache-auto-purge/wp-rocket-no-cache-auto-purge.php#L117) so that the cache is not cleared after the REST API access "POST /wp-json/wc/v2/products/categories/batch?".

Unfortunately without success.

Here is the snippet:

```
add_action( 'wp_rocket_loaded', function() {
    if ( defined( 'REST_REQUEST' ) ) {
	    $container = apply_filters( 'rocket_container', '' );
	    $container->get('event_manager')->remove_callback( 'create_term' , [ $container->get( 'purge_actions_subscriber' ), 'maybe_purge_cache_on_term_change'] );
	    $container->get('event_manager')->remove_callback( 'edit_term' , [ $container->get( 'purge_actions_subscriber' ), 'maybe_purge_cache_on_term_change'] );
	    $container->get('event_manager')->remove_callback( 'delete_term' , [ $container->get( 'purge_actions_subscriber' ), 'maybe_purge_cache_on_term_change'] );
    }
});
```

And here is an excerpt from the CallStack:

```
...
/wp-includes/rest-api.php(428): WP_REST_Server->serve_request()
/wp-includes/rest-api/class-wp-rest-server.php(439): WP_REST_Server->dispatch()
/wp-includes/rest-api/class-wp-rest-server.php(1063): WP_REST_Server->respond_to_request()
/wp-includes/rest-api/class-wp-rest-server.php(1230): WC_REST_Controller->batch_items()
/wp-content/plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-controller.php(269): WC_REST_Terms_Controller->update_item()
/wp-content/plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-terms-controller.php(506): wp_update_term()
/wp-includes/taxonomy.php(3405): do_action()
/wp-includes/plugin.php(517): WP_Hook->do_action()
/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters()
/wp-includes/class-wp-hook.php(326): WP_Rocket\Engine\Cache\PurgeActionsSubscriber->maybe_purge_cache_on_term_change()
/wp-content/plugins/wp-rocket/inc/Engine/Cache/PurgeActionsSubscriber.php(100): rocket_clean_domain()
/wp-content/plugins/wp-rocket/inc/functions/files.php(858): do_action()
...
```

I expect that after the REST API request "POST /wp-json/wc/v2/products/categories/batch?" the cache is not cleared.

**Comment by piotrbak** [comment]
Hello @GitHub-Mike could you contact our Support Team? We'll update the plugin helper then.

**Comment by GitHub-Mike** [comment]
> Hello @GitHub-Mike could you contact our Support Team? We'll update the plugin helper then.

Thank you. I will forward the request, but I cannot understand why this ticket is closed after 1.5 months without any reason or further information.
